readthedocs/templates/projects/import_config.html

```diff
@@ -0,0 +1,55 @@
+{% extends "projects/import_base.html" %}
+{% load i18n %}
+
+{% block content %}
+<h3>{% trans "Project configuration file (<code>.readthedocs.yaml</code>)" %}</h3>
+
+<p class="info">
+    {% blocktrans trimmed %}
+    Make sure your project has a <code>.readthedocs.yaml</code> at the root of your repository. This file is required by Read the Docs to be able to build your documentation. You can <a href="https://docs.readthedocs.io/en/stable/config-file/v2.html">read more about this in our documentation</a>.
+    {% endblocktrans %}
+</p>
+
+<p class="info">
+    Here you have an example for a common Sphinx project:
+
+    <pre class="small">
+        <code>
+# .readthedocs.yaml
```

Comment:
Because of the mix of whitespace styles on `<pre>` and a line break before the configuration file template starts, there is an empty line at the top/bottom of this block in display.

![image](https://github.com/readthedocs/readthedocs.org/assets/1140183/9154f994-a54d-48ef-b311-280092b884a0)


It makes for awkward HTML, but if you get rid of the whitespace between the first configuration file line start and the `<pre>`/`</pre>` tag, there won't be a blank line.